spec/defines/key_spec.rb

```diff
@@ -2,7 +2,8 @@
 
 describe 'apt::key', :type => :define do
   let(:facts) { { :lsbdistid => 'Debian' } }
-  GPG_KEY_ID = '4BD6EC30'
+  #GPG_KEY_ID = '4BD6EC30'
```

Comment:
Yes, this is a major pet peeve. Commented code adds _nothing_ unless its an example. Having Git means there's should be no such thing as "I'll comment this out in case I need it later."